\documentclass[a4paper,10pt]{article}
\usepackage{geometry}
\usepackage{enumitem}
\usepackage{fontawesome}
\usepackage{hyperref}
\usepackage{orcidlink}
\geometry{left=1.5cm,right=1.5cm,top=1.5cm,bottom=1.5cm}

% Command to create section titles
\newcommand{\sectiontitle}[1]{\vspace{10pt}\textbf{\Large #1}\vspace{5pt}\hrule\vspace{10pt}}

\begin{document}
	
	% Header section with name and contact information
	\begin{center}
		{\Huge \textbf{Gerald Amiel Ballena}} \\
		\vspace{3pt}
		\faEnvelope \ \href{mailto:gmballena@up.edu.ph}{gmballena@up.edu.ph} \quad
		\faGithub \ \href{https://github.com/GABallena}{github.com/GABallena} \quad
		\faLinkedin \ \href{https://linkedin.com/in/gerald-amiel-ballena}{linkedin.com/in/gerald-amiel-ballena} \\
		\vspace{3pt}
		\faMapMarker \ Metro Manila, Philippines \\
		\orcidlink{0009-0000-8857-9755} \ \href{https://orcid.org/0009-0000-8857-9755}{orcid.org/0009-0000-8857-9755}
	\end{center}
		
	
	% Summary section
	\sectiontitle{Summary}
	
	
	% Skills section
	\sectiontitle{Skills}
	\begin{itemize}[left=0pt]
		\item \textbf{Programming:} Python, Bash, R, Snakemake, Conda, LaTeX
		\item \textbf{Bioinformatics Tools:} 
		\item \textbf{Data Visualization:} \textit  ggplot2, Krona, Pavian, Matplotlib, Plotly, Shiny
		\item \textbf{Databases:} UniProt, CARD, NCBI, KEGG, RefSeq, SILVA, BUSCO
		\item \textbf{Operating Systems:} Ubuntu, Windows
		\item \textbf{Version Control:} Git, GitHub
	\end{itemize}
	
	% Experience section
	\sectiontitle{Experience}
	
	\textbf{Project Technical Specialist} \hfill \textit{University of the Philippines, College of Public Health} \\
	\textit{2024 â€“ Present}
	\begin{itemize}[left=0pt]
		\item Developed dynamic metagenomics pipelines for AMR detection.
		\item Spearheaded the creation of custom data visualization scripts using R and Python.
	\end{itemize}
	
	
	% Education section
	\sectiontitle{Education}
	
	\textbf{M.S.Microbiology} \hfill \textit{University of the Philippines Diliman} \\
	\textit{Graduated: 2022} \\
	Thesis: \textit{} \\
	Specifics: 
	
	\textbf{B.S Biology} \hfill \textit{University of the Philippines Baguio} \\
	\textit{Graduated: 2018} \\
	Thesis: \textit{} \\
	Specifics: 
	
	% Projects section
	\sectiontitle{Projects (See GitHub)}
	\textbf{Project4} \hfill \textit{GitHub: \href{https://github.com/GABallena/Project4}{Project4}}
	\begin{itemize}[left=0pt]
		\item most is NDA
		\item Identification of ARG-associated contigs via iterative alignment of reverse-translated CARD k-mers
	\end{itemize}
	
	\textbf{Automated Read Trimming and Quality Control Pipeline} \hfill \textit{GitHub: \href{https://github.com/GABallena/Project4}{Project4}}
	\begin{itemize}[left=0pt]
		\item Engineered a dynamic pipeline integrating \textbf{Trimmomatic}, \textbf{Fastp}, \textbf{BBDuk}, and \textbf{Cutadapt} with randomized trimming parameters across iterations, optimizing read preprocessing through extensive parameter testing.
		\item Automated \textbf{FastQC} assessments with retry logic for failed samples, ensuring robust quality control and traceability via \textbf{TSV logs} and \textbf{YAML-driven configurations}.
		\item Streamlined the workflow with comprehensive summary reporting on sample quality and trimming performance, enabling scalable, reproducible analyses in high-throughput genomic projects.
	\end{itemize}
	
	\textbf{Side} \hfill \textit{GitHub: \href{https://github.com/GABallena/Side}{Side}}
	\begin{itemize}[left=0pt]
		\item Pioneered an advanced \textbf{k-mer sequence space exploration pipeline}, enabling the detection of \textbf{biological signals} while effectively filtering out metagenomic noise to enhance data accuracy.
		\item Engineered \textbf{optimization of trimming parameters} using a \textbf{Metropolis-Hastings algorithm (MCMC)} to systematically refine high-dimensional genomic data for superior precision in downstream analyses.
		\item Implemented a \textbf{wavelet-based normalization strategy} on k-mer counts using \textbf{Continuous Wavelet Transform (CWT)}, ensuring robust signal processing and filtering of repetitive regions to spotlight high-coverage k-mers.
		\item Developed a high-throughput \textbf{automated k-mer extraction system} leveraging \textbf{nested dictionaries} and \textbf{pandas} to dynamically structure and organize k-mer counts by SCG, streamlining large-scale data management.
		\item Generated comprehensive \textbf{variance analysis reports}, driving deeper insights into the genomic landscape by identifying \textbf{conserved regions} and \textbf{highly variable loci}, propelling metagenomic research forward.
	\end{itemize}
	
	
	
		\textbf{Main} \hfill \textit{GitHub: \href{https://github.com/GABallena/Main}{Main}}
	\begin{itemize}[left=0pt]
		\item Developed an \textbf{automated version control system} for managing bioinformatics tools, dynamically tracking and updating software packages from the \textbf{Bioconda} repository to ensure up-to-date environments across projects.
		\item Leveraged the \textbf{Anaconda API} to automate the comparison of installed tool versions with the latest Bioconda releases, seamlessly updating Conda YAML files and improving reproducibility in bioinformatics pipelines.
		\item Built a framework for parsing \textbf{Conda and pip dependencies}, simplifying YAML file management by automating updates and reducing manual intervention in large-scale computational biology projects.
		\item Streamlined dependency management by ensuring the latest versions of bioinformatics tools are always available within wrapper tools, minimizing version conflicts and supporting efficient workflows.
	\end{itemize}
	


	
	% Publications section
	\sectiontitle{Publications}
	\begin{itemize}[left=0pt]
		\item \textbf{Ballena., et al.} 
	\end{itemize}
	
	% Certifications section
	\sectiontitle{Certifications}
	\begin{itemize}[left=0pt]
		\item Coursera
	\end{itemize}
	
	
	% Certifications section
	\sectiontitle{Certifications}
	\begin{itemize}[left=0pt]
		\item Nextflow for Bioinformatics, Coursera (2022)
		\item Machine Learning for Genomics, edX (2021)
		\item Data Visualization with Python (Plotly, D3.js), Udemy (2021)
		\item HPC for Bioinformatics, Coursera (2020)
	\end{itemize}
	
	% Certification of authenticity and signature
	\vspace{20pt}
	\noindent I hereby certify that the information provided above is true and correct to the best of my knowledge.
	
	\vspace{40pt}
	\noindent\makebox[2in]{\hrulefill} \\
	Gerald Amiel M. Ballena\\
	\textit{Signature}
	
	
\end{document}
